### DISCUSSION TITLE
Scaling Data

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2323] [Q&A] [answered]
I have manually scaled a dataframe including features in GLuonTS with a scaling rule I selected. Then I ran the training and forecasting process. Now I have a forecast_object from the package gluonts.model.forecast module and a "test object" where the comparison values are. But now I have  unnormalise this values again in order to draw the forecast and its evaluation. I wonder if anyone knows how to do this, i.e. how to manually unnormalise these two objects again before I produce error metrics and a drawing with the corresponding GluonTS functions.

**Comment by Msaleh87** [comment, marked as answer]
In your estimator, you could have could enable scaling by: `scaling = True`
You don't have to do this manually.

**Reply by MarleneBs** [reply]
Thank you, but this only works for the target value in deepar, does'nt it? For the features there is not possibility?
